monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+Lo bueno de este proceso de hashing es que la dirección de pago está especificada antes de la generación del hash, lo que permite crear hashes especificamente para alguien más. Los hashes, se crean utilizando la dirección de pago, y esta propiedad, es la que le permite a los pools de minería funcionar. Hay ciertas analogías a la minería de hashes en el mundo real. Es una creación que solo tiene valor al dueño de la dirección de destino y dura hasta que el siguiente bloque sea minado. Los bloques de Monero son minados cada dos minutos.
 
-For RPC-Pay, the hashes are calculated by the RPC client itself or by a (more powerful) helper to the client, and the hashes are sent to the server as they are calculated. Every hash sent counts as payment. A miniscule percentage of the hashes will actually have value to the server—those that, when represented as a number, mine a block by being small enough. These successful hashes occur at random within the many payment hashes. The server, then, counts all the hashes for credit, but only receives payment itself when it receives and publishes a successful mining hash on the Monero network.
+Para RPC-Pay, los hashes son calculados por el cliente RPC mismo o por un ayudante al cliente (con mayor potencia), y los hashes son enviados al servidor a medida que son calculados. Cada hash enviado cuenta como pago. Un porcentaje minúsculo de los hashes que, cuando se representen como un número y minen un bloque por ser lo suficientemente pequeños, eventualmente tendrán valor al servidor. Estos hashes exitosos ocurren de manera aleatoria dentro de los tantos pagos con los hashes. El servidor, luego, cuenta todos los hashes para crédito, pero solo recibe pago a si mismo cuando recibe y publica un hash exitoso en la red de Monero.
 
 ### _RPC-Pay_
 
-RPC-Pay is a new feature added to _monerod_ and wallets that enables payment for RPC calls using mining hashes. This capability is configured through the command line when starting the _monerod_ software. Representative startup parameters are illustrated below in [Example #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2). Similarly, there are commands with the CLI wallet that enable it to create hashes and pay monerod, and these are also presented in [Example #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2).
+RPC-Pay es una nueva característica añadida al _monerod_ y los monederos que habilitan pagos para las llamadas RPC, utilizando hashes de minería. Esta capacidad está configurada a través de la línea de comando al iniciar el software del _monerod_. Parámetros específicos de inicio se ilustran más abajo en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2). De manera similar, hay comandos con el monedero CLI que lo habilitan para crear hashes y pagarle al monerod. Estos también están representados en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2).
```

Comment:
The translation lost a little of the original meaning for the phrase, it would be like this in my opinion:

_RPC-Pay es una nueva característica añadida al _monerod_ **y a monederos que habilita** pagos para las llamadas RPC utilizando hashes de minería._

_**habilita**_ because it's speaking in singular of RPC-Pay as the function, not in plural for wallets.

Reply:
You're right. Adding the changes.

Reply:
Changed.